### ISSUE TITLE
A11y_User Management feature for Azure Data Studio_Login_LoginNew_AI4D:The Name property must not contain any characters in the private Unicode range.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#22842]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#SH_User Management feature for Azure Data Studio_Apr2023;#SQL Azure Data Studio ;#DesktopApp;#A11ySev3;#AI4D;#Win11;#Win32;#WCAG4.1.2;#Benchmark;#AINotInScope;#Closed;#Regressed:05-10-23

## Environment Details:
Application Name: Azure Data studio insiders
Version:1.44.0
Windows Version: Win11
Operating system : Windows 11 Enterprise (OS build22621.1555)

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
1. Open Azure data studio (Insiders) and connect with server with correct credentials
2. Tab till Security drop down and expand it
3. Tab till Logins Drop down and right click on it
4. Select New Login
5. Login - New (Preview) Dialog will open
6. Run Accessibility Insights for Web.

## Issue:
The following accessibility issue needs investigation.

**App:** azuredatastudio-insiders

**Element path:** text ''

**Issue Details:** The Name property must not contain any characters in the private Unicode range. [Section 508 502.3.1](https://www.access-board.gov/ict/#502-interoperability-assistive-technology)

## **How To Fix:** 
Modify the Name property, removing all characters in the range U E000 to U F8FF and replace them with meaningful, standard text content.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

### **Similar issue observed in following Scenario also.**
1. 'General_SQL Authentication'
2. Newly Created Login_Login - Access (Preview)
3. Users_User - New (Preview)
4. Users_User - test (Preview)

## User Impact:
Screen reader user will not able to correct information about control.

## Attachment:
![Similar issue Unique](https://user-images.githubusercontent.com/85863999/234181340-f7c3a094-c87e-492a-8f0f-a175dce88c34.png)
![AI Unique range](https://user-images.githubusercontent.com/85863999/234181341-caed2d18-a91e-4576-ae49-ccd94f1ecc23.png)

![Similar issue user new Unique](https://user-images.githubusercontent.com/85863999/234265094-96acc114-711b-4aed-9021-dee2678309d6.png)

**Comment by alanrenmsft** [comment]
@sspatil123 I think this is an issue with the accessibility insights, as you can see from the DOM tree, the aria-label of the dropdown doesn't have any unicode character, it is treating the ::after pseudo element as part of the name.

![image](https://github.com/microsoft/azuredatastudio/assets/13777222/b316381b-ee14-4157-9f60-49945ae294cb)

this is widely used in Azure Data Studio and VSCode.

I don't think this have any user impact. please close the issue. if you really think this is an issue, please open one in [vscode repo](https://github.com/microsoft/vscode) (ADS is forked from vscode) and you can use the tasks dropdown in screenshot below to reproduce the issue 
![image](https://github.com/microsoft/azuredatastudio/assets/13777222/20d70aa9-992e-414f-a1bf-8023633b4fba)

**Comment by sspatil123** [comment]
Screen reader is doing proper announcement. Hence closing bug
